scubagoggles/reporter/reporter.py

```diff
@@ -144,6 +145,38 @@ def build_report_html(fragments : list, product : str,
     html = html.replace('{{TABLES}}', collected)
     return html
 
+def build_report_json(tenant_domain : str, report_stats: dict,
+json_data: list) -> str:
+    '''
+    Adds data into JSON Template and formats the report accordingly
+
+    :param tenant_domain: the primary domain of the tenant.
+    :param main_report_name: dict containing the overall summary of the report tests.
+    :param json_data: list of json rego result output for specific baseline
+    '''
+    total_output = []
+    report_final = {}
+    now = datetime.now()
+    report_date = report_date = now.strftime('%Y-%m-%dT%H:%M:%SZ')
+
+    report_metadata = {
+        "TenantId":  None,
+        "DisplayName":  None,
+        "DomainName":  tenant_domain,
+        "Product":  "GWS",
```

Comment:
This key has been renamed
```suggestion
        "ProductSuite":  "GWS",
```

Also, there are two other keys that were recently added: `ProductsAssessed` and `ProductAbbreviation` mapping. See the schema spreadsheet for more details.